lib/workers/repository/process/fetch.ts

```diff
@@ -15,6 +15,9 @@ async function fetchDepUpdates(
 ): Promise<PackageDependency> {
   let dep = clone(indep);
   dep.updates = [];
+  if (dep.depName?.length === 0) {
+    dep.skipReason = 'empty';
```

Comment:
I think we need to introduce a more explanatory skip reason, like `missing-depname`